### PR TITLE
Added additional  props in button and dropdwon component

### PR DESCRIPTION
**Pull request by akate1**
- Added 'value' prop in Button and Dropdown-Item component
- Added 'active' prop in Dropdown-Item to identify active item
- Added direction to open menu in Dropdown

image

### CODE REVIEW

File: src/forms/dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -20,26 +20,40 @@ import {STOP_PROPAGATION} from ".";
 
 /**
  * DropDown Props
- * @param {label} label of checkbox
+ * @param {label} label of dropdown
  * @param {isNested} property to nest dropdown
  * @param {className} css property
  * @param {button} dropdown button
  * @param {showCaret} caret property
+ * @param {direction} direction to open a menu
```
Comment (psivakumaran) [inline src/forms/dropdown/Dropdown.tsx]: Small suggestion - More than 'direction to open menu' it should be 'position of the dropdown'

File: src/forms/dropdown/DropdownItem.tsx [hunk 2]
```diff
@@ -16,10 +16,12 @@ import {Button} from "../button";
 export type DropdownItemProps = {
     menuItemType: MenuItemType;
     label?: string;
+    value?: string;
     key?: string;
     isDisabled?: boolean;
     isExpandable?: boolean;
     isHeaderChild?: boolean;
+    active?: boolean;
```
Comment (psivakumaran) [inline src/forms/dropdown/DropdownItem.tsx]: Should this be named as 'isActive'?

Comment (akate1) [reply src/forms/dropdown/DropdownItem.tsx]: done

File: src/forms/dropdown/Dropdown.tsx (hunk 1 above)
Comment (akate1) [reply src/forms/dropdown/Dropdown.tsx]: We have used same naming convention in tooltip and signpost so keeping this as it is.

**Pull request merged**